weblate/locale/django.pot

```diff
@@ -2098,7 +2098,7 @@ msgid "Has been translated"
 msgstr ""
 
 #: weblate/checks/consistency.py:98
-msgid "This string has been translated in the past"
+msgid "This string has been translated in the past. Last Translation was \"%s\""
```

Comment:
This is not needed, the pot/po files are updated time to time by a dedicated script.